### DISCUSSION TITLE
Data source for ogbn-proteins node-label file

### DISCUSSION THREAD

**Discussion by lbalbi** [snap-stanford/ogb#332] [General]
When building the node-label matrix in the ogbn-proteins dataset, where did you retrieve the protein annotations to the labels/GO-terms from? I cannot seem to find a sizeable part of them in the GO .gaf files, even after converting (most of) the protein IDs from String to Uniprot. In case you did use a .gaf file, if you could also point out the version/date of the file used I'd be much obliged.

**Comment by lbalbi** [comment]
Hello, I was wondering if I could get any further information on the source of this part of your data? I see you've stated GO consortium as a data source, but we're having trouble finding them, could you confirm if you used a specific .gaf file from there to identify the protein-GO term links represented in node-label or if it was retrieved from another source? That would be very helpful.

**Reply by weihua916** [reply]
Hi! I believe you can find GO ID under `mapping/` directory of the dataset folder. Please ask [Marinka](https://dbmi.hms.harvard.edu/people/marinka-zitnik) for more information.
